cdap-ui/app/features/apps/templates/list.html

```diff
@@ -90,7 +90,17 @@
     <tbody>
       <tr ng-repeat="app in ListController.filtered = (ListController.apps | filter: ListController.searchText ) | myPaginate:ListController.currentPage | orderBy:sortable.predicate:sortable.reverse">
         <td >
+          <a ui-sref="apps.detail.overview.programs({ appId: app.name })"
+              ng-if="app.artifact.name !== ListController.GLOBALS.etlBatch && app.artifact.name !== ListController.GLOBALS.etlRealtime"
+              ng-click="MyOrderings.appClicked(app.name)">
+            <strong tooltip="{{app.name}}"
+                    tooltip-enable="app.name.length > 35"
+                    tooltip-append-to-body="true">
+              {{ ::app.name | myEllipsis:35 }}
+            </strong>
+          </a>
```

Comment:
It seems like we are going to be doing the check whether it is ETL apps quite a bit. Should we have a helper function in a service to check whether it is ETL apps or not?